rmf_gazebo_plugins/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,169 @@
+
+Below are the detailed rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`rmf_robot_sim_gazebo_plugins` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`rmf_robot_sim_gazebo_plugins` is at a stable version, i.e. `>= 1.0.0`.
+The current version can be found in its [package.xml](package.xml), and its change history can be found in its [CHANGELOG](CHANGELOG.rst).
+
+### Public API Declaration [1.iii]
+
+`rmf_robot_sim_gazebo_plugins` does not have a public API.
+
+### API Stability Policy [1.iv]
+
+`rmf_robot_sim_gazebo_plugins` does not have a public API.
+
+### ABI Stability Policy [1.v]
+
+`rmf_robot_sim_gazebo_plugins` does not have a public API.
+
+### API and ABI Stability Within a Released ROS Distribution [1.vi]
+
+`rmf_robot_sim_gazebo_plugins` does not have a public API.
+
+## Change Control Process [2]
+
+`rmf_robot_sim_gazebo_plugins` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-requirements).
+
+### Change Requests [2.i]
+
+`rmf_robot_sim_gazebo_plugins` requires that all changes occur through a pull request.
+
+### Contributor Origin [2.ii]
+
+`rmf_robot_sim_gazebo_plugins` does not require a confirmation of contributor origin.
+
+### Peer Review Policy [2.iii]
+
+All pull requests must have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull requests must pass CI on all platforms supported by RMF.
+The CI checks only that the package builds.
+The most recent CI results can be seen on [the workflow page](https://github.com/open-rmf/rmf_simulation/actions).
+
+### Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`rmf_robot_sim_gazebo_plugins` is documented in the [parent repository's README.md file](https://github.com/open-rmf/rmf_simulation/blob/master/README.md).
+
+### Public API Documentation [3.ii]
+
+`rmf_robot_sim_gazebo_plugins` does not have a public API.
+
+### License [3.iii]
+
+The license for `rmf_robot_sim_gazebo_plugins` is Apache 2.0, the type is declared in the [package.xml](package.xml) manifest file, and a full copy of the license is in the repository level [LICENSE](../LICENSE) file.
+
+### Copyright Statement [3.iv]
+
+The copyright holders each provide a statement of copyright in each source code file in `rmf_demo_tasks`.
+
+### Quality declaration document [3.v]
+
+This quality declaration is linked in the [README file](README.md).
```

Comment:
TODO

Reply:
The package-specific README.md does link to the quality declaration.